elixir-companies.toml

```diff
@@ -791,6 +792,14 @@ url = "https://kontomatik.com"
 github = "https://github.com/kontomatik"
 description = "We provide API to banks for proving client identity (KYC) and reading financial data (input for credit scoring, PFM, invoicing etc). We use Elixir for user panel/administration/business intelligence tools. Based in Warsaw, Poland"
 
+[[company]]
```

Comment:
Please add companies to the end of the list 👍

Reply:
It doesn't say so in the contribution instructions...
It seemed better to keep the list alphabetically ordered.
So I'll change it, and hopefully you add an instruction ;)

Reply:
We actually removed the instruction to put it in alphabetical order 😁

Reply:
Well, then an updated instruction seems useful. Saves you time, as well, in commenting on PRs.  ;)
And saves the contributors time, because they only need one commit.

Reply:
We've added a line to make sure there's no confusion 😁